packages/opentelemetry-exporter-collector/src/platform/node/utilWithJson.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as url from 'url';
+import * as http from 'http';
+import * as https from 'https';
+
+import { ReadableSpan } from '@opentelemetry/tracing';
+import * as collectorTypes from '../../types';
+import { toCollectorExportTraceServiceRequest } from '../../transform';
+import { CollectorTraceExporter } from './CollectorTraceExporter';
+
+export const DEFAULT_COLLECTOR_URL_JSON = 'http://localhost:55680/v1/trace';
+
+export function onInitWithJson(
+  _collector: CollectorTraceExporter,
+  _config: collectorTypes.CollectorExporterConfigNode
+): void {
+  // nothing to be done for json yet
+}
+
+export function sendSpansUsingJson(
+  collector: CollectorTraceExporter,
+  spans: ReadableSpan[],
+  onSuccess: () => void,
+  onError: (error: collectorTypes.CollectorExporterError) => void
+): void {
+  const exportTraceServiceRequest = toCollectorExportTraceServiceRequest(
+    spans,
+    collector
+  );
+
+  const body = JSON.stringify(exportTraceServiceRequest);
+  const parsedUrl = new url.URL(collector.url);
+
+  const options = {
+    hostname: parsedUrl.hostname,
+    port: parsedUrl.port,
+    path: parsedUrl.pathname,
+    method: 'POST',
+    headers: {
+      'Content-Length': Buffer.byteLength(body),
```

Comment:
What do you think about adding `Content-Type` header?

Reply:
added